src/_prerender_content.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="<%= htmlWebpackPlugin.files.chunks.prerender_content.entry %>"></script>
```

Comment:
* [ ] Still open: `storePrerenderedContent` should remove the fingerprinted `prerender_content.js` and `prerender_content.js.LICENSE.txt`.